##CRISPR

 CRISPR (/krspr/) (an acronym for clustered regularly interspaced short palindromic repeats) is a family of DNA sequences found in the genomes of prokaryotic organisms such as bacteria and archaea.[2] These sequences are derived from DNA fragments of bacteriophages that had previously infected the prokaryote. They are used to detect and destroy DNA from similar bacteriophages during subsequent infections. Hence these sequences play a key role in the antiviral (i.e. anti-phage) defense system of prokaryotes and provide a form of acquired immunity.[2][3][4][5] CRISPR are found in approximately 50% of sequenced bacterial genomes and nearly 90% of sequenced archaea.[6]


Diagram of the CRISPR prokaryotic antiviral defense mechanism[7]
Cas9 (or "CRISPR-associated protein 9") is an enzyme that uses CRISPR sequences as a guide to recognize and cleave specific strands of DNA that are complementary to the CRISPR sequence. Cas9 enzymes together with CRISPR sequences form the basis of a technology known as CRISPR-Cas9 that can be used to edit genes within organisms.[8][9] This editing process has a wide variety of applications including basic biological research, development of biotechnological products, and treatment of diseases.[10][11] The development of the CRISPR-Cas9 genome editing technique was recognized by the Nobel Prize in Chemistry in 2020 which was awarded to Emmanuelle Charpentier and Jennifer Doudna.[12][13]


Repeated sequences
The discovery of clustered DNA repeats took place independently in three parts of the world. The first description of what would later be called CRISPR is from Osaka University researcher Yoshizumi Ishino and his colleagues in 1987. They accidentally cloned part of a CRISPR sequence together with the "iap" gene (isozyme conversion of alkaline phosphatase) from the genome of Escherichia coli[14][15] that was their target. The organization of the repeats was unusual. Repeated sequences are typically arranged consecutively, without interspersed different sequences.[15][11] They did not know the function of the interrupted clustered repeats.

In 1993, researchers of Mycobacterium tuberculosis in the Netherlands published two articles about a cluster of interrupted direct repeats (DR) in that bacterium. They recognized the diversity of the sequences that intervened the direct repeats among different strains of M. tuberculosis[16] and used this property to design a typing method that was named spoligotyping, which is still in use today.[17][18]

Francisco Mojica at the University of Alicante in Spain studied repeats observed in the archaeal organisms of Haloferax and Haloarcula species, and their function. Mojica's supervisor surmised at the time that the clustered repeats had a role in correctly segregating replicated DNA into daughter cells during cell division because plasmids and chromosomes with identical repeat arrays could not coexist in Haloferax volcanii. Transcription of the interrupted repeats was also noted for the first time; this was the first full characterization of CRISPR.[18][19] By 2000, Mojica performed a survey of scientific literature and one of his students performed a search in published genomes with a program devised by himself. They identified interrupted repeats in 20 species of microbes as belonging to the same family.[20] Because those sequences were interspaced, Mojica initially called these sequences "short regularly spaced repeats" (SRSR).[21] In 2001, Mojica and Ruud Jansen, who were searching for additional interrupted repeats, proposed the acronym CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) to alleviate the confusion stemming from the numerous acronyms used to describe the sequences in the scientific literature.[19][22] In 2002, Tang, et al. showed evidence that CRISPR repeat regions from the genome of Archaeoglobus fulgidus were transcribed into long RNA molecules that were subsequently processed into unit-length small RNAs, plus some longer forms of 2, 3, or more spacer-repeat units.[23][24]

In 2005, yogurt researcher Rodolphe Barrangou, discovered that Streptococcus thermophilus, after iterative phage challenges, develops increased phage resistance, and this enhanced resistance is due to incorporation of additional CRISPR spacer sequences.[25] The Danish food company Danisco, which at that time Barrangou worked for, then developed phage resistant S. thermophilus strains for use in yogurt production. Danisco was later bought out by DuPont, which "owns about 50 percent of the global dairy culture market" and the technology went mainstream.[26]